CHANGELOG.md

```diff
@@ -67,7 +67,30 @@
   details.
 </details>
 
-## v1.32.0 - TBD
+## v1.33.0 - TBD
+
+## v1.32.0 - 2021-10
+
+### [BigQuery](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/README.md)
+
+* feat(generator): generate a SetIamPolicy() overload with an OCC loop ([#7276](https://github.com/googleapis/google-cloud-cpp/pull/7276))
```

Comment:
I do not think this change affected BigQuery?

Reply:
Oops.  Removed.  Thanks.